paderbox/utils/random_utils.py

```diff
@@ -177,6 +179,21 @@ def _sample(self, shape):
         return truncexpon(self.truncation / self.scale, self.loc, self.scale).rvs(shape)
 
 
+@dataclasses.dataclass
+class Choice(_Sampler):
+    events: int = None
```

Comment:
Why not simply `events: int`? (without a default value)
Then the `__post_init__``wouldn't be nessesary.